.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -49,17 +53,21 @@ body:
         - Ubuntu 20.04
         - Ubuntu 22.04
         - Ubuntu 24.04
+    validations:
+      required: true
+
   - type: textarea
     id: screenshot
     attributes:
       label: If relevant, including screenshots or a code block can be helpful in clarifying the issue.
-      placeholder: Code blocks begin and conclude by enclosing the code with three backticks (```) above and below it.
+      placeholder: "Code blocks begin and conclude by enclosing the code with three backticks (```) above and below it."
```

Comment:
Are the quotes included in the placeholder? If they are, I don't think a placeholder should have quotes

Reply:
works fine in yaml

<img width="811" alt="image" src="https://github.com/user-attachments/assets/f74b8c8b-91ad-45b0-b44c-fadd3c53451a">